### ISSUE TITLE
Add Show validation does not account for show timezone

### ISSUE THREAD

**Issue by paddatrapper** [LibreTime/libretime#1201]
**Describe the bug**
When creating a show from a browser in a timezone that is not the same as the server's timezone, the show start and end time validation uses the browser's timezone settings. This is different from the 'Now' and initial values populated in those fields.

**To Reproduce**
1. Set the server timezone to something behind the local timezone.
2. Go to calendar
2. Click 'New show'
3. Leave the autopopulated values for start and end (the current time in the server's timezone for start and an hour later for end)
4. Click 'Add this show'
5. See the validation error that the stand and end are in the past

**Expected behavior**
The start and end times should be validated according to the timezone selected for the show

**LibreTime version:**
d74d3db59a4a05a8330675046a7c1065e05f1735

**Installation method:**
 - OS: Debian
 - OS Version: Buster
 - Method: `./install -fap`
 - Details: `vagrant up debian-buster`

**Additional Notes**

For this I had my browser in UTC+2, while the server was UTC

**Comment by jooola** [comment]
Have to admit after digging in the JS code, I prefer not to touch it as it seems really fragile.

**Comment by paddatrapper** [comment]
Yeah, I'm in the same boat! I think this is something to deal with with the Vue transition
